### PR TITLE
Demo project

### PR DESCRIPTION
**Pull request by alex6171**

### CODE REVIEW

**Comment by alex6171** [comment]
Demo

**Comment by maios** [comment]
Dudeee, make the commit messages a bit descriptive -.-

**Comment by maios** [comment]
http://www.commitlogsfromlastnight.com/

File: Demo/ViewController.swift [hunk 1]
```diff
@@ -9,18 +9,45 @@
 import UIKit
 import BadgifySwift
 
-class ViewController: UIViewController {
+fileprivate let swiftColor = UIColor(red: 223/255, green: 92/255, blue: 67/255, alpha: 1)
```
Comment (maios) [inline Demo/ViewController.swift]: this can be put inside class scope with `private` access

File: Demo/ViewController.swift [hunk 2]
```diff
@@ -9,18 +9,45 @@
...
+    
+    @IBAction func segmentedControlValueChanged(_ sender: UISegmentedControl) {
+        switch sender.selectedSegmentIndex {
+            case 0:
+                self.badgeView.badgeValue = 1
+                self.stepper.isHidden = false
+            case 1:
+                self.badgeView.badgeImage = #imageLiteral(resourceName: "cc_logo")
+                self.stepper.isHidden = true
+            default: break
```
Comment (maios) [inline Demo/ViewController.swift]: you can omit `self.`

File: Demo/ViewController.swift [hunk 3]
```diff
@@ -9,18 +9,45 @@
...
+    private let badgeView: BadgeView = {
+        let view = BadgeView()
+        view.badgeValue = 1
+        view.backgroundColor = .white
+        view.textColor = swiftColor
+        view.badgeFont = .systemFont(ofSize: 14)
+        view.minBadgeSize = CGSize(width: 25, height: 25)
+        view.xOffset = 5
+        view.yOffset = 5
+        return view
```
Comment (maios) [inline Demo/ViewController.swift]: hey for demo let's use the default values

**Comment by maios** [comment]
If the bug you told me about is that we cannot set the badge value to zero using UIStepper is because the minimumValue of stepper is currently set to 1, you can set it to 0

File: Demo/ViewController.swift (hunk 3 above)
Comment (alex6171) [reply Demo/ViewController.swift]: I think it's good to point out you can do that by using the properties. Your call though 😀

**Comment by alex6171** [comment]
It wasn't that 😀
Run demo project and select image for the badge. Image I bundled is fairly big for the badge, 48*48, so it stretches the badge to be the same size as the image
Not sure if bug or a feature, but I'd go with having the option to select contentMode for the image and resize the badge accordingly

**Review by maios — APPROVED** [review]

**Pull request merged**